lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -614,6 +624,91 @@ export function detectTestingLibraryUtils<
       );
     };
 
+    /**
+     * Determines whether a given node is some reportable `act` util.
+     *
+     * An `act` is reportable if some of these conditions is met:
+     * - it's related to Testing Library module (this depends on Aggressive Reporting)
+     * - it's related to React DOM Test Utils
+     */
+    const isActUtil = (node: TSESTree.Identifier): boolean => {
+      const isTestingLibraryAct = isPotentialTestingLibraryFunction(
+        node,
+        (identifierNodeName, originalNodeName) => {
+          return [identifierNodeName, originalNodeName]
+            .filter(Boolean)
+            .includes('act');
```

Comment:
```suggestion
          return [identifierNodeName, originalNodeName]
            .includes('act');
```

Minor suggestion, but I think you can skip the filtering as using includes should be enough (all falsey values are different from `'act'`)

Reply:
Yeah, although I need to refactor all this mechanism in a separate PR as mentioned in [this comment](https://github.com/testing-library/eslint-plugin-testing-library/pull/365/files#r626109283). So not sure if worth it, I just added for consistency with the other `is*Util` methods to be honest.